aviary/interface/methods_for_level2.py

```diff
@@ -1178,11 +1178,14 @@ def add_post_mission_systems(self, include_landing=True):
         A user can override this with their own postmission systems.
         """
 
+        if self.pre_mission_info['include_takeoff'] and self.mission_method is HEIGHT_ENERGY:
+            self._add_post_mission_takeoff_systems()
```

Comment:
should include_takeoff be in post_mission_info, since the takeoff system is added to post mission

Reply:
I vote no. We only add the takeoff in the postmission because of a model ordering quirk. I wouldn't expect a user to think "ah yeah takeoff occurs after the mission" or have to think about it like that.